src/client/ProjectFinder.ts

```diff
@@ -87,7 +111,7 @@ export async function findMetaEdProjects(): Promise<WorkspaceProjects> {
       ...newProjectMetadata(folderPath),
       projectName: projectJsonMetadata.projectName,
       projectVersion,
-      namespaceName,
+      namespaceName: namespaceName!,
```

Comment:
It's fine in tests, but **never, never, never** suppress null errors with ! in code.